CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## 0.5.3
+
+### Enhancements
+
+- Allow conflicting extras in explicit index assignments ([#9160](https://github.com/astral-sh/uv/pull/9160))
```

Comment:
This is a huge thing, do we want to highlight it and link the docs page?

Reply:
That's a good idea. I'll add a blurb to the top.